hack/test_e2e.sh

```diff
@@ -282,7 +289,10 @@ function install_kpng {
     # TODO move this to ci:
     # docker load --input kpng-image.tar
     echo "loading kpng:test docker image"
-    kind load docker-image kpng:test --name ${E2E_CLUSTER_NAME}
+    CMD_KIND_LOAD_KPNG_TEST_IMAGE=("kind load docker-image kpng:test --name ${E2E_CLUSTER_NAME}")
+    ${CMD_BUILD_IMAGE}
```

Comment:
why do we need `CMD_BUILD_IMAGE` here? If I understand correctly, that's part of `docker_build` function right?
https://github.com/kubernetes-sigs/kpng/blob/8496fd72894dc6e887e8d0242f8cb6ba32572b6b/hack/test_e2e.sh#L86-L100

Reply:
Good catch, probably a buffer in my copy/past. Thanks!